blocks/navigation/navigation.js

```diff
@@ -72,7 +71,6 @@ export default class Navigation extends ComponentBase {
   async setupCompactedNav() {
     if (!this.navCompactedContentInit) {
       this.navCompactedContentInit = true;
-      await component.multiLoadAndDefine(['accordion', 'icon']);
```

Comment:
How will these components be loaded if we are manually creating the elements?

Reply:
They are dependencies at the componentList

Reply:
<img width="443" alt="Screenshot 2024-09-27 at 17 53 36" src="https://github.com/user-attachments/assets/a77e1a5e-543a-45c2-a27f-084fcd5d8e78">

Reply:
And with that, we can also add the order of loading by prio

Reply:
But in some cases we might not need to load these until it's required, like `accordion` is only needed in mobile.

And for `popup-trigger`... the `popup` component module is only needed if someone actually opens a pop-up.

Maybe we should keep the option to load and define a component only when needed.

Reply:
```
loadModules([], componentList.accordion)
```